src/scripts/safari/SafariPokemonList.ts

```diff
@@ -38,7 +38,7 @@ class SafariPokemonList {
         const pokemon : SafariType[] = SeededRand.shuffleArray(App.game.party.caughtPokemon.map((p) => p.name)
             .filter((p) => !PokemonHelper.hasEvableLocations(p) && Object.keys(PokemonHelper.getPokemonLocations(p)).length)
             .map((p) => {
-                return {name: p, weight: 3};
+                return {name: p, weight: 10};
```

Comment:
Was this change on purpose? It looks unrelated.

Reply:
This change was intentional, I should have mentioned it initially.
Filler mons in the Friend Safari were put there to keep it from breaking on strange challenge runs, but they were showing up way more frequently than felt good. This change was made to make the intended mons (Only EVable in the Friend Safari) much more common, and the filler mons be a side note.